ui/app/mainui/AppMain.qml

```diff
@@ -1180,8 +1179,11 @@ Item {
         sourceComponent: StatusEmojiPopup {
             directParent: appMain.Window.window.contentItem
             height: 440
-            settings: appMainLocalSettings
+            recentEmojis: appMainLocalSettings.recentEmojis
+            skinColor: appMainLocalSettings.skinColor
             emojiModel: SQUtils.Emoji.emojiModel
+            onSetSkinColor: color => appMainLocalSettings.skinColor = color
+            onSetRecentEmojis: recentEmojis => appMainLocalSettings.recentEmojis = recentEmojis
```

Comment:
I think it'd be cleaner to actually have a small `Settings {}` inside the popup and encapsulate the `skinColor` and `recentEmojis` entirely inside the popup. I'd consider that way cleaner than it is now; some of those settigns are handled inside the popup (in the opened/closed handlers), and some outside here